packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -758,6 +787,155 @@ module.exports = (
           });
         }
 
+        return {};
+      },
+    },
+    {
+      method: 'POST',
+      path: '/recovery_email/secondary/resend_code',
+      options: {
+        auth: {
+          strategy: 'sessionToken',
+        },
+        validate: {
+          payload: {
+            email: validators.email().required(),
```

Comment:
Same as the question above, why pass in an email here, is it to support multiple secondaries? If we only support one secondary, couldn't we by default send the email there and avoid relying on user generated input?